### PR TITLE
Support default storage class in helm chart

### PR DESCRIPTION
**Pull request by n0rad**

### CODE REVIEW

**Comment by n0rad** [comment]
signed

File: helm/provisioner/templates/provisioner.yaml [hunk 1]
```diff
@@ -146,6 +146,12 @@ apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
   name: {{ $val.name }}
+  {{- if kindIs "map" $val.storageClass }}
+  {{- if $val.storageClass.isDefaultClass }}
```
Comment (msau42) [inline helm/provisioner/templates/provisioner.yaml]: Thanks for the PR! Can you add a test for this under https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/tree/master/helm/examples

File: helm/provisioner/values.yaml [hunk 2]
```diff
@@ -76,6 +76,8 @@ classes:
   # Uncomment to create storage class object and configure it.
   # storageClass:
     # reclaimPolicy: Delete # Available reclaim policies: Delete/Retain, defaults: Delete.
+    # isDefaultClass: true # set as default class
```
Comment (msau42) [inline helm/provisioner/values.yaml]: Ah I forgot, can you also update the helm/README.md with the new option as well?

Comment (n0rad) [reply helm/provisioner/values.yaml]: done, but tests failed and do not look like linked to my changes

**Comment by msau42** [comment]
/lgtm
/approve

Thanks!

**Pull request merged**